packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -1470,7 +1468,7 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, () => {
 			return await makeRestApiRequest(
 				rootStore.restApiContext,
 				'POST',
-				`/workflows/${startRunData.workflowData.id}/run?partialExecutionVersion=${version}`,
+				`/workflows/${startRunData.workflowData.id}/run?partialExecutionVersion=${settingsStore.partialExecutionVersion}`,
```

Comment:
@cstuncsik do you have an idea how I can write a test for this? If not necessary let me know.

Reply:
I see the `settingsStore.partialExecutionVersion` computed is already tested we could do a kind of a duplication here, checking if the API call has the correct partial execution version number when `runWorkflow` calls it